cyences_app_for_splunk/default/eventtypes.conf

```diff
@@ -60,3 +60,9 @@ search = sourcetype="o365:management:activity" Workload=AzureActiveDirectory Ope
 search = sourcetype="gapps:report:login" events{}.name IN ("login_success", "login_failure")
 # Note - For other events like login_challenge with passed status google is also sending login_success events, 
 # so we don't have to assign authentication tag to those events
+
+#######################
+#### Windows DNS Logs
+######################
+[dns_dc_queries]
```

Comment:
I've worked on DNS stuff before for Chapman

```
[dns_dc_queries]
search = index=msad sourcetype="MSAD:NT6:DNS"

[dns_isc_queries]
search = index=dns (sourcetype="isc:bind:query" OR sourcetype="isc:bind:queryerror")
```

for tags.conf
```
[eventtype=dns_dc_queries]
network = enabled
resolution = enabled
dns = enabled

[dns_isc_queries]
network = enabled
resolution = enabled
dns = enabled
```

Reply:
Here I am intentionally not adding index. But I will add the other one.